proposals/collection-expression-arguments.md

```diff
@@ -354,7 +355,9 @@ d = [with(capacity: 2)]; // new Dictionary<string, int>(capacity: 2)
 r = [with(capacity: 2)]; // error: 'capacity' parameter not recognized
 ```
 
-If the target type is any other type, and the *argument list* is not empty, a binding error is reported.
+If the target type is any other type, then:
+* If the *argument list* is empty, the *argument list* has no effect.
```

Comment:
i would actually remove this.  and we can make a note that `with()` for an array/span is not allowed.

--

Generally speaking, i think we should just bring empty `with()` as its own discussion question :)

Reply:
Removed.